src/octopeer-github/content/ElementSelectionBehaviour/NameElementSelectionBehaviour/OtherContributerNameElementSelectionBehaviour.ts

```diff
@@ -1,17 +1,15 @@
-/// <reference path="NameElementSelectionBehaviour.ts"/>
+/// <reference path="../AbstractElementSelectionBehaviour.ts"/>
 /**
  * Created by Mathias on 2016-05-11.
  * Class for selecting the OtherContributer name
  */
-class OtherContributerNameElementSelectionBehaviour extends NameElementSelectionBehaviour {
+class OtherContributerNameElementSelectionBehaviour extends AbstractElementSelectionBehaviour {
 
     /**
      * Constructor of this class.
      * @param database The database that is used to store the Elements.
      */
     public constructor(database: DatabaseAdaptable) {
-        super(database);
-        this.elementDsc = "teststss";
-        this.elementID = new ElementID(305);
+        super(database, new ElementID(305), "teststss");
```

Comment:
While you're at it, maybe add a `// TODO` at all the "tesetstststst" things so we can search the source folder for todo's (or use WebStorm's TODO viewer) :)